fuzz/fuzz_targets/deserialize_transaction.rs

```diff
@@ -58,7 +60,7 @@ mod tests {
     #[test]
     fn duplicate_crash() {
         let mut a = Vec::new();
-        extend_vec_from_hex("00", &mut a);
+        extend_vec_from_hex("04000000000000000000", &mut a);
```

Comment:
I thought we weren't leaving this in upstream code.

Reply:
Oops, I did not mean to check in an edit to the test.